### ISSUE TITLE
To list files stored in self hosted storage

### ISSUE THREAD

**Issue by EduMelo** [keeweb/keeweb#2026]
That's not exactly a feature request it is more a doubt. Shouldn't self hosted keeweb allow me to access the internal stored files when I acess it from a different browser than the one I usually do?

What is happening is that when I try this all I get is a input box asking for an file that is on my local file system, but when I save the file, I synced it to server file system

Did I get this functionallity wrong?

**Comment by edkedk99** [comment]
I am not sure if that's what you asked, but you need to pass the url for you file on every device you connect.

**Comment by EduMelo** [comment]
That's right. I thought that keeweb would store the file. I didn't understand that  is not the way it works
